src/role_scout.js

```diff
@@ -9,6 +9,7 @@
 roles.scout = {};
 roles.scout.settings = {
   layoutString: 'M',
+  amount: [1],
```

Comment:
hum strange, scouts spawned well when i made it. but with your other PR checks modified i supose it is no more a problem